functions.php

```diff
@@ -136,3 +132,49 @@ function govwind_register_button_styles()
     );
 }
 add_action('init', 'govwind_register_button_styles');
+
+/**
+ * Auto register block patterns in /patterns folder 
+ */
+function govwind_register_all_block_patterns()
```

Comment:
As discussed. This feels odd to have as wordpress already import patterns. I think we need understand what is causing the pattern not showing up on search. Happy to leave in for now though until we get to bottom of this issue.

Reply:
I agree and will take the 🪓 to it.